sdk/resourcemanager/Azure.ResourceManager.Core/src/ResourceGroupContainer.cs

```diff
@@ -57,7 +57,7 @@ private ResourceGroupsOperations Operations
         /// <param name="managedBy"> Who the resource group is managed by. </param>
         /// <returns> A builder with <see cref="ResourceGroup"/> and <see cref="ResourceGroupData"/>. </returns>
         /// <exception cref="ArgumentNullException"> Location cannot be null. </exception>
-        public ArmBuilder<ResourceGroupResourceIdentifier, ResourceGroup, ResourceGroupData> Construct(LocationData location, IDictionary<string, string> tags = default, string managedBy = default)
+        public ResourceGroupBuilder Construct(LocationData location, IDictionary<string, string> tags = default, string managedBy = default)
```

Comment:
I wonder if we should keep the ResourceGroupBuilder class.  Since we don't have the abstraction, and don't plan to update the builder concept until after the preview, seems like we can drop these classes and the properties that return them.  Adding these methods after preview, when the Builder concept is fully fleshed would be a non-breaking change